core/src/test/scala/com/nvidia/spark/rapids/tool/qualification/PluginTypeCheckerSuite.scala

```diff
@@ -160,7 +160,7 @@ class PluginTypeCheckerSuite extends FunSuite with Logging {
 
   test("supported operator score from dataproc") {
     val checker = new PluginTypeChecker("dataproc")
-    assert(checker.getSpeedupFactor("FilterExec") == 2.8)
+    assert(checker.getSpeedupFactor("FilterExec") >= 2.0)
```

Comment:
wait I'm confused by this, why are we allowing >?  I get that we have different numbers based on csp but I think we should test them specifically.

Reply:
I'm working on another PR to more generally avoid static checks of exec speedup factors for the execs that will be updated through auto-generated PRs.

Reply:
This is the replacement PR: https://github.com/NVIDIA/spark-rapids-tools/pull/87